src/test/java/org/jenkinsci/plugins/durabletask/PowershellScriptTest.java

```diff
@@ -246,6 +225,8 @@
     }
 
     @Test public void noStdoutPollution() throws Exception {
+        // TODO? Binary wrapper does not allow redirection of non-success streams to the logger
```

Comment:
So Powershell can only re-direct the non-success streams (`2` - `6`) to a separate file (each stream must have a separate file), or all to the `success` stream (`1`). Additionally, the default behavior is that these non-error streams get merged into the `success` stream when a script is launched via `-File` or `-Command`. I don't see a clean way of separating `verbose, warning, debug, etc` from the `success` stream and putting them into the log. I could mirror anything in output to be put into the log, but I think that defeats the purpose of this test. I'm comfortable with documenting the behavior of the binary as diverging from the script wrapper for this particular case as it is the only deviation I know of.

I believe `powershellhelper.ps1` was able to get by this restriction by having special conditions, such as catching errors and then writing them to log, but I don't think it's worth that complexity.

If anyone has any suggestions, I'm more than open to hear them.

Reply:
We rely pretty heavily on having a clean stdout as we have often generate a JSON payload within the script and Write-Output to pass it back up. So with that we utilize the verbose stream for logging during the execution to know what's going on.

Reply:
I think I managed a clean stdout now. Unfortunately, through `go`, the non-error streams were redirected to stdout, but if I did I piped the output to a file, I could get a clean output. Unfortunately that required some bom-less file writing tricks, but I think it's good now.